src/reducers/ui/settings.js

```diff
@@ -39,9 +43,11 @@ function create(defaultState = initState) {
   return handleActions({
     [a.profile.toggleTab]: (state, { payload }) => ({ ...state, currentProfileTab: payload }),
     [a.tools.toggleTab]: (state, { payload }) => ({ ...state, currentToolsTab: payload }),
+    [a.account.toggleTab]: (state, { payload }) => ({ ...state, currentAccountTab: payload }),
```

Comment:
Hmm... I also missed the point at which we started include UI actions / reducers into **topcoder-react-lib**, but it is not correct: unlike actions / reducers related to communication with Topcoder APIs, that can, and should be effectively reused between different Topcoder apps, UI-related actions / reducers belong only to the apps that need them, thus should not be placed into **topcoder-react-lib**.

Reply:
Hmm.. i think so. The first committer of this file is not me. so i just follow existing code to add new function for this part and keep consistent

Reply:
OK. I'll move ui actions / reducers to community-app, and put back removed error messages